test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/Components/TestRunnerBase.razor

```diff
@@ -143,10 +143,9 @@
         _testRenderFragments[methodName] = fragment;
     }
 
-    protected async Task WaitForMapToRender([CallerMemberName] string methodName = "")
+    protected async Task WaitForMapToRender([CallerMemberName] string methodName = "", int tries = 100)
```

Comment:
Nice! I was just thinking of the fact that we needed a way to adjust the timeout for different tests. I think the WebMap tests could use longer timeouts as well.

Reply:
Although, it would be nice to make it a clear timespan timeout instead of a `tries` property. Could we make the new parameter `timeoutInSeconds` or `timeoutInMilliseconds` and then do a conversion in this method to create the number of tries? Then the caller could just say `await WaitForMapToRender(timeoutInSeconds: 60)`.